package.json

```diff
@@ -25,39 +25,39 @@
     "node": ">=6.0"
   },
   "dependencies": {
-    "http-status": "1.4.2",
-    "nock": "12.0.3",
+    "http-status": "1.5.0",
+    "nock": "13.0.6",
     "node-fetch": "2.6.1",
-    "uuid": "8.1.0"
+    "uuid": "8.3.2"
   },
   "devDependencies": {
-    "@sinonjs/fake-timers": "6.0.1",
-    "@types/chai": "4.2.11",
-    "@types/chai-as-promised": "7.1.2",
-    "@types/express": "4.17.6",
-    "@types/mocha": "7.0.2",
-    "@types/node-fetch": "2.5.7",
-    "@types/sinon": "9.0.4",
-    "@types/sinon-chai": "3.2.4",
-    "@types/uuid": "8.0.0",
-    "@typescript-eslint/eslint-plugin": "3.2.0",
-    "@typescript-eslint/eslint-plugin-tslint": "3.2.0",
-    "@typescript-eslint/parser": "3.2.0",
+    "@sinonjs/fake-timers": "7.0.2",
+    "@types/chai": "4.2.14",
+    "@types/chai-as-promised": "7.1.3",
+    "@types/express": "4.17.11",
+    "@types/mocha": "8.2.0",
+    "@types/node-fetch": "2.5.8",
+    "@types/sinon-chai": "3.2.5",
+    "@types/sinonjs__fake-timers": "6.0.2",
```

Comment:
What the difference to `"@sinonjs/fake-timers": "7.0.2",`?

Reply:
I just learned this when having a look at koa...this is the way you name `@types` packages which are already scoped (`@sinonjs` in this case - yeah, that is not nice...)

Reply:
These are the types - sinon is currently implementing types on their own, but are not completely done yet - see https://github.com/sinonjs/fake-timers#help-us-get-our-typescript-definitions-production-ready

Reply:
Therefore also the distinct commit